server/src/main/java/org/elasticsearch/search/fetch/FetchPhase.java

```diff
@@ -438,13 +437,17 @@ private void prepareNestedHitContext(FetchSubPhase.HitContext hitContext,
         return nestedIdentity;
     }
 
-    private void loadStoredFields(SearchShardTarget shardTarget, LeafReaderContext readerContext, FieldsVisitor fieldVisitor, int docId) {
+    private void loadStoredFields(SearchShardTarget shardTarget,
```

Comment:
do we want to make this function `static`?

Reply:
👍